spec/queries/district_analytics_query_spec.rb

```diff
@@ -109,7 +110,7 @@
     end
 
     describe '#registered_patients_by_period' do
-      it "should'nt count discarded patients" do
+      it "shouldn't count discarded patients" do
```

Comment:
Consider "doesn't" or "excludes" to avoid the use of "should"

Reply:
Addressed [here](https://github.com/simpledotorg/simple-server/pull/888/commits/8abca6e2ad04e2b11778ae36777db676215f4f06).